packages/liferay-npm-scripts/src/scripts/test.js

```diff
@@ -51,10 +50,8 @@ module.exports = function(arrArgs = []) {
 			);
 		}
 
-		withBabelConfig(() => {
-			spawnSync('jest', ['--config', CONFIG_PATH, ...arrArgs.slice(1)], {
-				env
-			});
+		spawnSync('jest', ['--config', CONFIG_PATH, ...arrArgs.slice(1)], {
+			env
```

Comment:
Is this change legit? I would expect that we probably have tests in liferay-portal that won't work properly without our standard Babel config as a base.

Reply:
It probably breaks everything :muscle: I wanted to get rid if `withBabelConfig` since it is used only there, but seeing that we need it, I may move it to that module, for example...

I don't like the idea of leaving a `withBabelConfig` hanging around in its own module since someone may use it for build purposes and break `deployFast` again... What do you think?

Reply:
Maybe just do it locally here without re-instating `withBabelConfig`. That is probably better than making it `withBabelConfig__DO_NOT_USE_OR_YOU_WILL_BE_FIRED`.